appsec/tests/integration/src/test/www/symfony62/src/Controller/HomeController.php

```diff
@@ -21,13 +21,12 @@ public function homeAction(Request $request)
         );
     }
 
-    /**
-     * @Route("/dynamic-path/{param01}", name="dynamic-path")
-     */
-    public function dynamicAction(Request $request)
+    #[Route("/dynamic-path/{param01}", locale: "en")]
+    #[Route("/caminho-dinamico/{param01}", locale: "pt")]
```

Comment:
How is the PT route reported if locale is `pt`. I can't see any test

Reply:
We don't report the routes and I don't hit this route, only then `en` one. The point is that there being several locales forces the route name not be a key in the route collection; instead we get `route_name.locale`.